### DISCUSSION TITLE
>50 amps

### DISCUSSION THREAD

**Discussion by btarb24** [emporia-vue-local/esphome#178] [Q&A] [answered]
new to ESPHome and curious as to options for monitoring a 60 amp 2pole circuit that has asymmetric load.  The load is an air handler with resistive heat coils that reports an mca of 55a.

Here are some ideas.  Can anyone shed some light on the feasibility of them and perhaps the choice they'd go with and why?

1. using a 3rd 200 amp CT in the top 3.5mm ports that would typically be for a 3rd phase.
2. just using one of the standard 50a clamps and assuming the load will remain under 50 amps.  mca has a safety buffer of 25%, right?  Uncertain if these clamps even have a true 50 amp saturation limit or if it's slightly higher?
3. I will probably buy and link two emporias. My understanding is the 2nd will not need to clamp the mains .. does that allow for those 3.5s to be used as normal monitors for high load circuits?
4. giving up and using a separate clamp for each phase - i assume i can combine their values in software?

**Comment by flaviut** [comment, marked as answer]
All of these options will work just fine. The software side of things would be something like

```
  - platform: template
    name: "Circuit Name Power"
    lambda: return id(cir2).state + id(cir4).state;
    update_interval: 1s
    id: total_power
    unit_of_measurement: "W"
```
